examples/client-and-server-coexistence/main.c

```diff
@@ -0,0 +1,46 @@
+/**
+ * This application intends to verify that the symbols from the
+ * client-side and server-side SDKs do not clash, thus enabling both SDKs to be
+ * used within the same application.
+ */
+
+#include <launchdarkly/client_side/bindings/c/sdk.h>
+#include <launchdarkly/server_side/bindings/c/sdk.h>
+
+#include <launchdarkly/client_side/bindings/c/config/builder.h>
+#include <launchdarkly/server_side/bindings/c/config/builder.h>
+
+#include <launchdarkly/bindings/c/context_builder.h>
+
+int main() {
+    LDContextBuilder context_builder = LDContextBuilder_New();
+    LDContextBuilder_AddKind(context_builder, "user", "example-user-key");
+    LDContextBuilder_Attributes_SetName(context_builder, "user", "Sandy");
+    LDContext context = LDContextBuilder_Build(context_builder);
+
+    LDClientConfigBuilder client_config_builder =
+        LDClientConfigBuilder_New("foo");
+    LDClientConfig client_config;
```

Comment:
I think the linter would like these to be assigned to null, which I can get behind.

Reply:
I'll update. The Build method sets it to `nullptr` on failure, which is why I didn't do it originally.